application/src/test/java/org/thingsboard/server/controller/WidgetTypeControllerTest.java

```diff
@@ -190,6 +190,20 @@ public void testGetBundleWidgetTypes() throws Exception {
         Collections.sort(loadedWidgetTypes, idComparator);
 
         Assert.assertEquals(widgetTypes, loadedWidgetTypes);
+
+        loginCustomerUser();
+
+        List<WidgetType> loadedWidgetTypes2 = doGetTyped("/api/widgetTypes?widgetsBundleId={widgetsBundleId}",
+                new TypeReference<>(){}, widgetsBundle.getId().getId().toString());
+        Collections.sort(loadedWidgetTypes2, idComparator);
+        Assert.assertEquals(widgetTypes, loadedWidgetTypes2);
+
+        List<WidgetTypeDetails> loadedWidgetTypes3 = doGetTyped("/api/widgetTypesDetails?widgetsBundleId={widgetsBundleId}",
```

Comment:
Using numbers in the name is a bad practice.
loadedWidgetTypes3 - > loadedWidgetTypeDetailsCustomer

Reply:
Please, add a similar test for `/api/widgetTypesDetails` above for the tenant_admin